README.md

```diff
@@ -1,2 +1,71 @@
 # uphold/litecoind
 A litecoind docker image.
+
+[![uphold/litecoind][docker-pulls-image]][docker-hub-url] [![uphold/litecoind][docker-stars-image]][docker-hub-url] [![uphold/litecoind][docker-size-image]][docker-hub-url] [![uphold/litecoind][docker-layers-image]][docker-hub-url]
+
+## Supported tags and respective `Dockerfile` links
+- `0.10.4.0-alpine`, `0.10.4-alpine` ([0.10.4/alpine/Dockerfile](https://github.com/uphold/docker-litecoin/blob/master/0.10.4/alpine/Dockerfile))
+
+## What is litecoind?
+_from [litecoinwiki](https://litecoin.info/Litecoin)_
+
+litecoind is a program that implements the Litecoin protocol for remote procedure call (RPC) use. For more information see https://github.com/litecoin-project/litecoin
+
+## Usage
+### How to use this image
+This image contains the main binaries from the Litecoin Core project - `litecoind`, `litecoin-cli` and `litecoin-tx`. It behaves like a binary, so you can pass any arguments to the image and they will be forwarded to the `litecoind` binary:
+
+```sh
+$ docker run --rm -it uphold/litecoind \
+  -printtoconsole \
+  -regtest=1 \
+  -rpcallowip=172.17.0.0/16 \
+  -rpcpassword=bar \
+  -rpcuser=foo
+```
+
+By default, `litecoind` will run as as user `litecoin` for security reasons and with its default data dir (`~/.litecoin`). If you'd like to customize where `litecoind` stores its data, you must use the `LITECOIN_DATA` environment variable. The directory will be automatically created with the correct permissions for the `litecoin` user and `litecoind` automatically configured to use it.
+
+```sh
+$ docker run --env LITECOIN_DATA=/var/lib/litecoind --rm -it uphold/litecoind \
```

Comment:
No need for interactive tty mode here (`-it`). Also, use shorthand `-e` argument for consistency.